recipes/phinx.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * Phinx recipe for Deployer
+ *
+ * @author    Alexey Boyko <ket4yiit@gmail.com>
+ * @copyright 2016 Alexey Boyko 
+ * @license   MIT https://github.com/deployphp/recipes/blob/master/LICENSE
+ *
+ * @link https://github.com/deployphp/recipes
+ *
+ * @see http://deployer.org
+ * @see https://phinx.org
+ */
+
+/**
+ * Get Phinx command
+ *
+ * @return Path to Phinx
+ */
+env(
+    'phinx_path', function () {
+        $isExistsCmd = 'if [ -f %s ]; then echo true; fi';
+
+        try {
+            $phinxPath = run('which phinx')->toString();
+        } catch (\RuntimeException $e) {
+            $phinxPath = null;
+        }
+
+        if ($phinxPath !== null) {
+            return "phinx";
+        } else if (run(
+            sprintf(
+                $isExistsCmd,
+                '{{release_path}}/vendor/bin/phinx'
+            )
+        )->toBool()
+        ) {
+            return "{{release_path}}/vendor/bin/phinx";
+        } else if (run(
+            sprintf(
+                $isExistsCmd,
+                '~/.composer/vendor/bin/phinx'
+            )
+        )->toBool()
+        ) {
+            return '~/.composer/vendor/bin/phinx';
+        } else {
+            throw new \RuntimeException(
+                'Cannot find phinx. 
+            Please specify path to phinx manually'
+            );
+        }
+    }
+); 
+
+/**
+ * Make Phinx command from env options 
+ * 
+ * @param string $cmdName Name of command
+ * @param array  $conf    Command options(config)
+ *
+ * @return string Phinx command to execute
+ */
+set(
+    'phinx_get_cmd', function ($cmdName, $conf) {
+        $phinx = env('phinx_path');
```

Comment:
Now env functionality moved to set/get pair. You can find more all UPGRADE.md file.